lib/simplecov_linter_formatter/formatters/source_file_formatter.rb

```diff
@@ -0,0 +1,34 @@
+module SimpleCovLinterFormatter
+  class SourceFileFormatter
+    FILE_COLUMN = 1
+
+    def initialize(source_file)
+      @source_file = source_file
+    end
+
+    def format
+      lines.map do |line|
+        build_message(line)
+      end
+    end
+
+    private
+
+    def build_message(line)
+      [
+        @source_file.filename,
+        line.line_number,
+        FILE_COLUMN,
+        "#{line.status}-#{lines_count}",
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Style/TrailingCommaInArrayLiteral: Avoid comma after the last item of an array.